### PR TITLE
Adding conformance results for django oci!

### PR DESCRIPTION
**Pull request by vsoch**
So excited for this! 

https://opencontainers.org/posts/blog/2021-11-30-oci-conformance/

Here are the results for Django OCI. if y'all aren't familiar with Django, it's a popular Python framework that has a really nice module/plugin system, so django-oci is implemented as such. A made this a little while back hoping to bring in more academics to the OCI community!

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

### CODE REVIEW

**Review by tianon — APPROVED** [review]
Seems sane to me!

**Comment by jdolitsky** [comment]
@vsoch - any build badges to add? (can do so later too)

**Comment by vsoch** [comment]
The docs said badges should go into the live folder, which as I understand it, is for live registries? Django-OCI is a plug-in for others to use for their apps. Could I put them under v1.0 instead?

**Comment by jdolitsky** [comment]
@vsoch - if you have a badge, for example, that tests the `main` / `master` branch of django-oci, you could add it under distribution-spec/live/

Check out the zot example (also opensource):
https://github.com/opencontainers/oci-conformance/tree/main/distribution-spec/live/zot

**Comment by vsoch** [comment]
okay added!

File: distribution-spec/v1.0/django-oci/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+vendor: vsoch
```
Comment (jdolitsky) [inline distribution-spec/v1.0/django-oci/PRODUCT.yaml:1]: terrific vendor!

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**